emission/pipeline/model_stage.py

```diff
@@ -11,7 +11,7 @@
 import emission.net.usercache.abstract_usercache as enua
 import emission.storage.timeseries.abstract_timeseries as esta
 import emission.storage.decorations.tour_model_queries as esdtmq
-from emission.core.get_database import get_db, get_mode_db, get_section_db
+# from emission.core.get_database import get_db, get_mode_db, get_section_db
```

Comment:
Is this one not useful in the model_stage?
Trying to see if I can add any comment to a closed PR.

Reply:
Looks like you can!
If you search for `get_db`, `get_mode_db` etc in this file, you don't find any matches. So it is not used any more. This has also bitrotted - as you can see from the blame (https://github.com/e-mission/e-mission-server/blame/20aac1f52204bd8af402f0cb63c917648836d60f/emission/core/get_database.py) it was there many years ago but has been removed now